### DISCUSSION TITLE
Limit RMM User Accounts to only see certain Clients?

### DISCUSSION THREAD

**Discussion by modem-19** [amidaware/tacticalrmm#1718] [Q&A]
Is it possible in Tactical RMM, to setup a new user account (say user2, whereas I'm user1), where user2 can only view 2 of the 10 clients listed in RMM?  So that user2 can manage the clients of those 2 clients, yet me as user1, can see all 10 clients and manage those 10 clients computers as needed?

I've seen this in other RMM's like ConnectWise, but wasn't sure if this was possible with Tactical.

This isn't a make or break feature request for me, I'm still demoing it and coming up with lots of questions.

**Comment by silversword411** [comment]
Yes, use the permission manager.

![2023-12-28_021837 - brave_Tactical_RMM_-_Brave](https://github.com/amidaware/tacticalrmm/assets/13395348/f9a9270f-e789-49dc-bd7b-2734c366e125)

Right now, there is one caveat <https://docs.tacticalrmm.com/mesh_integration/#security-implications>
